stack-orchestration/manual-stacks/dns-zones/template.yaml

```diff
@@ -0,0 +1,144 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: >-
+  All the DNS Hosted Zones for the Orchestration Stubs.
+Parameters:
+  Environment:
+    Description: The name of the environment to deploy to
+    Type: String
+    AllowedValues:
+      - dev
+      - build
+      - staging
+      - integration
+      - production
+
+Conditions:
+  IsProduction: !Equals [ !Ref Environment, production]
+
+Mappings:
+  EnvironmentConfiguration:
+    dev:
+      rpDomainName: rp-dev.build.stubs.account.gov.uk
+      docAppRpDomainName: doc-app-rp-dev.build.stubs.account.gov.uk
+    build:
+      rpDomainName: rp-build.build.stubs.account.gov.uk
+      docAppRpDomainName: doc-app-rp-build.build.stubs.account.gov.uk
+    staging:
+      rpDomainName: rp-staging.build.stubs.account.gov.uk
+      docAppRpDomainName: doc-app-rp-staging.build.stubs.account.gov.uk
+    integration:
+      rpDomainName: rp-integration.build.stubs.account.gov.uk
+      docAppRpDomainName: doc-app-rp-integration.build.stubs.account.gov.uk
+    production:
+      rpDomainName: rp.stubs.account.gov.uk
+      docAppRpDomainName: doc-app-rp.build.stubs.account.gov.uk
```

Comment:
In the readme it says records/certifications will be generated for `<subdomain>.<environment>.stubs.account.gov.uk` - should this be the same structure here? (`.` rather than `-`)